src/model/object/entity/individual/fred/fred.h

```diff
@@ -7,7 +7,7 @@
 
 class Fred : public Individual {
 public:
-    Fred(double x, double y, double energy);
```

Comment:
I'd prefer to actually call the variable "angle", because a is very non-descriptive

Reply:
I will merge `direction`/`angle` and `visionRange` into one vector called `vision` anyways.

Reply:
I will put those changes in the vector PR (#30

Reply:
_if_ we make the direction random, this will vanish anyways